pravega/pravega
# Issue 5852: Speed up build verification

**Change log description**  
Reduce build times by running tests in parallel.
Fix many flakey test, and other test issues.

**Purpose of the change**  
Attempts to parallelize where possible
Fixes #5852

**What the code does**  
This mainly affect tests and build process. A few patches that were causing build failures are included.
This fixes #5864, #5008, #5657, #5515, #3691, #5565, #4966. And many others not previously reported.

**How to verify it**  
We need to actually run several builds to make sure this works, and does not introduce race conditions, ooms etc.
